runtime/common/src/constants.rs

```diff
@@ -35,3 +46,144 @@ impl sp_std::fmt::Debug for MaxDataSize {
+pub const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
+
+/// We allow for 0.5 of a second of compute with a 12 second average block time.
+pub const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND / 2;
+
+pub type ZERO = ConstU32<0>;
+pub type FIFTY = ConstU32<50>;
+
+pub type FrameSystemMaxConsumers = ConstU32<16>;
+pub type MsaMaxKeys = ConstU8<25>;
+pub type MsaMaxProviderNameSize = ConstU32<16>;
+
+parameter_types! {
+	pub const SchemasMaxRegistrations: SchemaId = 65_000;
+}
+pub type SchemasMinModelSizeBytes = ConstU32<8>;
+pub type SchemasMaxBytesBoundedVecLimit = ConstU32<65_500>;
+
+parameter_types! {
+	pub const VestingPalletId: PalletId = PalletId(*b"py/vstng");
+}
+
+parameter_types! {
+	pub const MinVestedTransfer: Balance = 0;
+}
+
+pub type ORMLMaxVestingSchedules = FIFTY;
+
+parameter_types! {
+	pub const MinimumPeriod: u64 = SLOT_DURATION / 2;
+}
+
+pub type AuthorshipUncleGenerations = ZERO;
+
+parameter_types! {
+	pub const ExistentialDeposit: Balance = EXISTENTIAL_DEPOSIT;
+}
+
+pub type BalancesMaxLocks = FIFTY;
+pub type BalancesMaxReserves = FIFTY;
+pub type SchedulerMaxScheduledPerBlock = FIFTY;
+
+pub type PreimageMaxSize = ConstU32<{ 4096 * 1024 }>;
+
+parameter_types! {
+	pub const PreimageBaseDeposit: Balance = 1 * MILLIUNIT;
+	pub const PreimageByteDeposit: Balance = 1 * MICROUNIT;
+}
+pub type CouncilMaxProposals = ConstU32<25>;
+
+parameter_types! {
+	pub const CouncilMotionDuration: BlockNumber = 5 * DAYS;
+}
+
+pub type TCMaxProposals = ConstU32<25>;
+pub type TCMaxMembers = ConstU32<3>;
+
+parameter_types! {
+	pub const TCMotionDuration: BlockNumber = 5 * DAYS;
+}
+
+// Config from
+// https://github.com/paritytech/substrate/blob/367dab0d4bd7fd7b6c222dd15c753169c057dd42/bin/node/runtime/src/lib.rs#L880
+parameter_types! {
+	pub const LaunchPeriod: BlockNumber = 28 * DAYS;
+	pub const VotingPeriod: BlockNumber = 28 * DAYS;
+	pub const FastTrackVotingPeriod: BlockNumber = 3 * DAYS;
+	pub const EnactmentPeriod: BlockNumber = 30 * DAYS;
+	pub const CooloffPeriod: BlockNumber = 28 * DAYS;
+	pub const MinimumDeposit: Balance = 100 * UNIT;
+}
+
+pub type DemocracyMaxVotes = ConstU32<100>;
+pub type DemocracyMaxProposals = FIFTY;
+
+pub type TransactionPaymentOperationalFeeMultiplier = ConstU8<5>;
+
+parameter_types! {
+	/// Relay Chain `TransactionByteFee` / 10
+	pub const TransactionByteFee: Balance = 10 * MICROUNIT;
+}
+
+parameter_types! {
+	pub const ReservedXcmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT / 4;
+	pub const ReservedDmpWeight: Weight = MAXIMUM_BLOCK_WEIGHT / 4;
+}
+
+pub type SessionPeriod = ConstU32<{ 6 * HOURS }>;
+pub type SessionOffset = ZERO;
+
+pub type AuraMaxAuthorities = ConstU32<100_000>;
+
+pub type CollatorMaxCandidates = ZERO;
+pub type CollatorMinCandidates = ZERO;
+
+parameter_types! {
+	pub const CollatorPotId: PalletId = PalletId(*b"PotStake");
+	// pub const CollatorSessionLength: BlockNumber = 6 * HOURS;
+	pub const CollatorExecutiveBody: BodyId = BodyId::Executive;
```

Comment:
@shannonwells I confirmed that this is an unused copy-pasta from statemint. So it can go away along with the xcm dependency.